scripts/deployment_manager/configs/darwin.json

```diff
@@ -0,0 +1,106 @@
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_transformations.dylib",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_preproc.so",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_ir_reader.so",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_onnx_reader.so",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_c_api.dylib",
+        "deployment_tools/inference_engine/lib/intel64/libAutoPlugin.so",
+        "deployment_tools/inference_engine/lib/intel64/plugins.xml",
+        "deployment_tools/inference_engine/external/tbb",
+        "deployment_tools/ngraph/lib"
+      ]
+    },
+    "cpu": {
+      "ui_name": "Inference Engine Runtime for Intel(R) CPU",
+      "group": ["ie"],
+      "dependencies" : ["ie_core"],
+      "files": [
+        "deployment_tools/inference_engine/lib/intel64/libMKLDNNPlugin.so"
+      ]
+    },
+    "vpu": {
+      "ui_name": "Inference Engine Runtime for Intel(R) Movidius(tm) VPU",
+      "group": ["ie"],
+      "dependencies" : ["ie_core"],
+      "files": [
+        "deployment_tools/inference_engine/lib/intel64/libmyriadPlugin.so",
+        "deployment_tools/inference_engine/lib/intel64/myriad_compile",
+        "deployment_tools/inference_engine/lib/intel64/myriad_perfcheck",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_legacy.dylib",
+        "deployment_tools/inference_engine/lib/intel64/usb-ma2x8x.mvcmd",
+        "deployment_tools/inference_engine/lib/intel64/pcie-ma2x8x.mvcmd"
+      ]
+    },
+    "opencv": {
+      "ui_name": "OpenCV",
+      "group": ["opencv"],
+      "dependencies" : [],
+      "files": [
+        "opencv/version.txt",
+        "opencv/setupvars.sh",
+        "opencv/lib",
+        "opencv/bin"
+      ]
+    },
+    "python_common": {
+      "group": ["python"],
+      "dependencies" : ["ie_core"],
+      "files": [
+        "python/python3",
+        "python/requirements.txt"
+      ]
+    },
+    "python3.6": {
+      "ui_name": "OpenVINO Python API for Python3.6",
+      "group": ["python"],
+      "dependencies" : ["ie_core", "python_common"],
+      "files": [
+        "python/python3.6"
+      ]
+    },
+    "python3.7": {
+      "ui_name": "OpenVINO Python API for Python3.7",
+      "group": ["python"],
+      "dependencies" : ["ie_core", "python_common"],
+      "files": [
+        "python/python3.7"
+      ]
+    },
+    "python3.8": {
+      "ui_name": "OpenVINO Python API for Python3.8",
+      "group": ["python"],
+      "dependencies" : ["ie_core", "python_common"],
+      "files": [
+        "python/python3.8"
+      ]
+    },
+    "python3.9": {
+      "ui_name": "OpenVINO Python API for Python3.9",
+      "group": ["python"],
+      "dependencies" : ["ie_core", "python_common"],
+      "files": [
+        "python/python3.9"
+      ]
```

Comment:
@slyubimt how this will be aligned with new idea with wheels? 

Do we need to have deployment manager support for python at all?

Reply:
Till we not switched to 'new idea with wheels' - let's migrate DM code AS IS.